MMM-Fuel.js

```diff
@@ -277,10 +308,31 @@ Module.register("MMM-Fuel", {
         appendTo.appendChild(row);
 
         if(this.config.showAddress){
+			function titleCase(str) {
+				words = str.toLowerCase().split(/-| /);
+
+				for(var i = 0; i < words.length; i++) {
+				var letters = words[i].split('');
+				letters[0] = letters[0].toUpperCase();
+				words[i] = letters.join('');
+				}
+				return words.join(' ');
+			}
             var details = document.createElement("tr");
             details.setAttribute("colspan", 2 + this.config.types.length + (this.config.open ? 1 : 0));
+            var city_string = data.place.charAt(0).toUpperCase() + data.place.toLowerCase().slice(1);
+            var street_string = ((titleCase(data.street)).replace("Straße", "Str.")).replace("Duesseldorf","Düsseldorf");
+			var streetno_string = "";
```

Comment:
as mentioned earlier here are static values like Düsseldorf which only affect you, this should be removed or dynamic